site/content/en/docs/manual/advanced/ai-tools.md

```diff
@@ -1,155 +1,309 @@
-Supported DL models are not bound to the label and can be used for any objects.
-To create a polygon usually you need to use regular or positive points.
-For some kinds of segmentation negative points are available.
-Positive points are the points related to the object.
-Negative points should be placed outside the boundary of the object.
-In most cases specifying positive points alone is enough to build a polygon.
-A list of available out-of-the-box interactors is placed below.
+See:
 
-- Before you start, select the `magic wand` on the controls sidebar and go to the `Interactors` tab.
-  Then select a label for the polygon and a required DL model. To view help about each of the
-  models, you can click the `Question mark` icon.
+- [AI Tools](#ai-tools)
+  - [Interactors](#interactors)
+  - [Interactors Models](#interactors-models)
+  - [Detectors](#detectors)
+  - [Detectors models](#detectors-models)
+  - [Trackers](#trackers)
+  - [Tracker models](#tracker-models)
+- [OpenCV](#opencv)
+  - [Intelligent scissors](#intelligent-scissors)
+  - [Histogram Equalization](#histogram-equalization)
+  - [TrackerMIL](#trackermil)
 
-  ![](/images/image114_detrac.jpg)
+## AI Tools
 
-- Click `Interact` to enter the interaction mode. Depending on the selected model,
-  the method of markup will also differ.
-  Now you can place positive and/or negative points. The [IOG](#inside-outside-guidance) model also uses a rectangle.
-  Left click creates a positive point and right click creates a negative point.
-  After placing the required number of points (the number is different depending on the model),
-  the request will be sent to the server and when the process is complete a polygon will be created.
-  If you are not satisfied with the result, you can set additional points or remove points.
-  To delete a point, hover over the point you want to delete, if the point can be deleted,
-  it will enlarge and the cursor will turn into a cross, then left-click on the point.
-  If you want to postpone the request and create a few more points, hold down `Ctrl` and continue (the `Block`
-  button on the top panel will turn blue), the request will be sent after the key is released.
+**AI tools** are available from the [controls sidebar](/docs/manual/basics/controls-sidebar/).
+<br>Click ![Magic wand](/images/image189.jpg) to start annotate.
 
-  ![](/images/image188_detrac.jpg)
+### Interactors
 
-- In the process of drawing, you can select the number of points in the polygon using the switch.
+Use interactors, to create a polygon semi-automatically.
 
-  ![](/images/image224.jpg)
+Supported DL models are not bound to the label, you can use them to track any object.
+
+When creating a polygon, you can use positive (regular) points
+or negative points (for some models):
+
+- Positive points are the points related to the object.
+- Negative points should be placed outside the object.
+
+![](/images/image188_detrac.jpg)
```

Comment:
Positive points can be anywhere on the object. But you describe here how Deep Extreme Cut algorithm expects points. 

The algorithm utilizes so-called extreme points: https://arxiv.org/pdf/1711.09081.pdf
```
This paper explores the use of extreme points in an object (left-most, right-most, top,
bottom pixels) as input to obtain precise object segmentation for images and videos. We
do so by adding an extra channel to the image in the input of a convolutional neural
network (CNN), which contains a Gaussian centered in each of the extreme points. The
CNN learns to transform this information into a segmentation of an object that matches
those extreme points.
```